apps/daffio/src/app/docs/shared/services/docs.service.spec.ts

```diff
@@ -40,7 +40,19 @@ describe('DaffioDocService', () => {
     expect(req.request.method).toEqual('GET');
 
     req.flush(doc);
-  });
+	});
+
+  it('should be able to retrieve a doc when there is a fragment in the url', () => {
+    service.get('my/path#fragment').subscribe((apiDoc) => {
+      expect(apiDoc).toEqual(doc);
+    });
```

Comment:
can you not use `jasmine-marbles` for this expectation?

Reply:
changed